design/one-pager-mrds.md

```diff
@@ -0,0 +1,248 @@
+# Managed Resource Definitions
+
+* Owner: Nic Cope (@negz)
+* Reviewer: ???
+* Status: Draft
+
+## Background
+
+Crossplane managed resources (MRs) suffer from two long-standing issues.
+
+MRs can automatically write sensitive connection details like addresses,
+usernames, and passwords to a Kubernetes Secret. We call these connection
+details. Today it's very hard to discover what connection details an MR
+supports. The only way to do so is to create an MR and see what it writes to its
+connection secret. This is tracked in [issue 1143][1] - Crossplane's third most
+upvoted issue.
+
+Crossplane bundles related MRs together into a provider - like provider-github
+or provider-aws-ec2. Today some of these providers install up to 100 MRs. With
+v2 this'll double, because v2 introduces namespaced MRs but keeps the existing
+cluster scoped MRs for backward compatibility.
+
+Each MR is powered by a CustomResourceDefinition (CRD) and a controller. Each
+CRD has a performance penalty on the Kubernetes API server - mostly memory
+usage. Each CRD also creates one or more API endpoints that clients like kubectl
+may need to walk.
+
+It's not possible to install only some of a provider's MRs. It's all or nothing.
+The ability to install only the MRs you intend to use is tracked in [issue
+2869][2] and [issue 4192][3] - collectively Crossplane's most upvoted issue of
+all time. A few years ago we broke up the largest providers into [families of
+smaller providers][4]. This helped alleviate the issue, but some large
+Crossplane deployments still work around it - for example by building their own
+providers that have only the MRs they need.
+
+## Proposal
+
+I propose we introduce a new type - ManagedResourceDefinition (MRD).
```

Comment:
Today Providers don't require Crossplane to be installed to run, but this change would couple providers to crossplane as they would no longer work independently, they need crossplane running to get their CRD installed. Which also means their controllers need to be much smarter around starting informers to not crash loop.